activerecord/lib/active_record/core.rb

```diff
@@ -641,17 +641,22 @@ def strict_loading?
     #
     # ==== Parameters
     #
-    # * value - Boolean specifying whether to enable or disable strict loading.
-    # * mode - Symbol specifying strict loading mode. Defaults to :all. Using
-    #          :n_plus_one_only mode will only raise an error if an association
-    #          that will lead to an n plus one query is lazily loaded.
+    # * +value+ - Boolean specifying whether to enable or disable strict loading.
+    # * <tt>:mode</tt> - Symbol specifying strict loading mode. Defaults to :all. Using
```

Comment:
I think for symbols `+:symbol+` works fine for RDoc [[ref](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font)].

There's also a few symbols missing fixed-font, so it'd be nice if those were consistent.

Reply:
I'm just cargo culting the `<tt>`. I have no idea what it does 😄 . It might be good to document this as well in the API contributing guides.

Reply:
Ah, I see it's already documented :).

Reply:
Yeah this is super handy:

```
echo "+:to_param+" | rdoc --pipe
# => <p><code>:to_param</code></p>
```